crates/key-value/src/host_component.rs

```diff
@@ -0,0 +1,28 @@
+use spin_core::HostComponent;
+
+use crate::{Config, KeyValueDispatch};
+
+pub struct KeyValueComponent {
```

Comment:
I wonder if we are overloading the term `Config` here or whether we want to move away from using it

Reply:
Happy to change it if desired.  Do you have an alternative in mind?

Reply:
the [`Config` enum](https://github.com/fermyon/spin/blob/2cab1dab9ea0e0fbfc622e9e186039c1d2feaf14/crates/key-value/src/sqlite.rs#L12) could be `DatabaseLocation`

Reply:
Good idea -- thanks!